roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -153,7 +153,7 @@ rbac_enabled: "{{ 'RBAC' in authorization_modes or kubeadm_enabled }}"
 
 ## List of key=value pairs that describe feature gates for
 ## the k8s cluster.
-kube_feature_gates: []
+kube_feature_gates: ['Initializers=true']
```

Comment:
every keys in default/main.yml should also be added in examples/group_vars/k8s-cluster.yml